web/src/components/Container.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Container = ({ children }) => (
+  <div className="site-main">
+    <div className="container p2">{children}</div>
+  </div>
+);
+
+Container.propTypes = {
+  children: PropTypes.node.isRequired
+};
+
+export default Container;
```

Comment:
What's the purpose of this `Container` element?  I see it's used by the login page, but nowhere else?

Reply:
so `Container` should be used when there's no sidebar, so it's basically the one-column layout wrapper (which so far is only really the login page)